### ISSUE TITLE
Add hashrate watchdog

### ISSUE THREAD

**Issue by kusayuzayushko** [NebuTech/NBMiner#427]
Add a commandline option to restart/reboot/run restart script if reported hashrate drops below some level.

Requested because on some GPUs I'm experiencing sudden hashrate drops.
![image](https://user-images.githubusercontent.com/521312/132684729-72c550cd-806c-41e9-af4a-b7d4b3524244.png)
 
is it a LHR kicks in or is it something else - I'm not sure, but the hashrate watchgod would help me a lot.

**Comment by mspecdesign** [comment]
yes i was just wondering this, would be a great backup to reboot the miner if hash rate is below X for X mins.
i have had a few missed LHR locks that would have been recovered by this. NbMiner 39.7, 3060Ti , win 10, 496.49 drivers,
